src/Components/Web.JS/.gitignore

```diff
@@ -1,2 +1,3 @@
 ﻿node_modules/
 dist/Debug/
+dist/Release/blazor.webassembly.js
```

Comment:
AFAIK this isn't part of the shared framework \ source build, so it doesn't need to be checked in. Correct me if I'm wrong.